var/spack/repos/builtin/packages/pism/package.py

```diff
@@ -114,30 +117,3 @@ def cmake_args(self):
     def setup_run_environment(self, env):
```

Comment:
By the way, `setup_run_environment()` should not be needed.

Reply:
You suggest removing the entire block?
```
def setup_run_environment(self, env):
    env.set("PISM_PREFIX", self.prefix)
    env.set("PISM_BIN", self.prefix.bin)
```

Reply:
I do. PISM does not use these environment variables.